pokepro/src/components/Background/Background.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import pokeball from '../../assets/assets/pokeball.png'
+import pokeball2 from "../../assets/assets/pokeball2.png"
+import "./background.css"
+
+function Background() {
+  return (
+    <div className='background'>
+      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
+      <img src={pokeball2} alt="pokeball2" className="pokeball pokeball2"/>'
+      <img src={pokeball2} alt="pokeball2" className="pokeball pokeball-2"/>'
+      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
+      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+     
+      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
+      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
+    </div>
+  )
+}
```

Comment:
We can shorten it like this.

```suggestion
const Background = () =>  (
    <div className='background'>
      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
      <img src={pokeball2} alt="pokeball2" className="pokeball pokeball2"/>'
      <img src={pokeball2} alt="pokeball2" className="pokeball pokeball-2"/>'
      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
     
      <img src={pokeball2} alt="pokeball" className="pokeball pokeball-2"/>'
      {/* <img src={pokeball} alt="pokeball" className="pokeball pokeball-1"/>' */}
    </div>
  )
```